### PR TITLE
Fix datepikcer

### PR DESCRIPTION
**Pull request by jsp-up**

### CODE REVIEW

**Review by ydomenjoud — CHANGES_REQUESTED** [review]
Add unit tests to check this fix.

**Review by ydomenjoud — APPROVED** [review]

File: projects/kalidea/kaligraphi/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kalidea/kaligraphi",
-  "version": "0.0.53",
+  "version": "0.0.54",
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/package.json]: why ?

**Review by ydomenjoud — CHANGES_REQUESTED** [review]
why did you update version ?

File: projects/kalidea/kaligraphi/package.json (hunk 1 above)
Comment (jsp-up) [reply projects/kalidea/kaligraphi/package.json]: It wasan error

**Review by ydomenjoud — APPROVED** [review]

**Pull request merged**